### PR TITLE
MEN-3192 02.Overview/12.Taxonomy/docs.md

### PR DESCRIPTION
**Pull request by merlin-northern**
ChangeLog:none
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

File: 02.Overview/12.Taxonomy/docs.md [hunk 1]
```diff
@@ -5,7 +5,76 @@ taxonomy:
     label: reference
 ---
 
-# Migration notes
-~~~~~~~~~~~~~~~~~~~~
-* NOTE: Likely to be added after the reorganization
-~~~~~~~~~~~~~~~~~~~~
+This section presents the basic terms used throughout the documentation.
+
+* _Pending device_ - A device that already sent an authorization request to the Mender server,
```
Comment (drewmoseley) [inline 02.Overview/12.Taxonomy/docs.md:10]: A device that has already sent an authorization request to the Mender server without admission through preauthorization or user authorization.
(not sure about the terms at the end of the sentence though)

File: 02.Overview/12.Taxonomy/docs.md [hunk 2]
```diff
@@ -5,7 +5,76 @@ taxonomy:
...
+but user has not yet performed any action on it.
+
+* _Rejected device_ - A device that already sent a first authorization request to the Mender server,
+and user has performed "reject" operation on it.
```
Comment (drewmoseley) [inline 02.Overview/12.Taxonomy/docs.md:14]: A device that has already sent an authorization request to the Mender server, which has been explicitly rejected by the user.

File: 02.Overview/12.Taxonomy/docs.md [hunk 3]
```diff
@@ -5,7 +5,76 @@ taxonomy:
...
+
+* _Authorization set_ - A key assigned to a device. It can be in one of
+the following states:
+  * "rejected"
+  * "accepted"
+  * "pending"
+  * "preauthorized"
+  
+* _Pre authorized device_ - A device given by authorization set, that will 
+shift into "accepted" state at the first time when it requests authorization.
```
Comment (drewmoseley) [inline 02.Overview/12.Taxonomy/docs.md:45]: shift into "accepted" state automatically when it requests authorization.

File: 02.Overview/12.Taxonomy/docs.md [hunk 4]
```diff
@@ -5,7 +5,76 @@ taxonomy:
...
+
+* _Binary delta_ - Binary patch for a filesystem image.
+
+* _Board integration_ - The low-level integration required to get Mender working on
+a board (like Raspberry Pi3).
```
Comment (drewmoseley) [inline 02.Overview/12.Taxonomy/docs.md:50]: I'm not sure how we spell "Raspberry Pi3" elsewhere. I think there is a space between Pi and 3.  But I think you can just drop the parenthesized expression.

Comment (merlin-northern) [reply 02.Overview/12.Taxonomy/docs.md]: sure

File: 02.Overview/12.Taxonomy/docs.md (hunk 3 above)
Comment (merlin-northern) [reply 02.Overview/12.Taxonomy/docs.md]: sure

File: 02.Overview/12.Taxonomy/docs.md (hunk 2 above)
Comment (merlin-northern) [reply 02.Overview/12.Taxonomy/docs.md]: sure

File: 02.Overview/12.Taxonomy/docs.md (hunk 1 above)
Comment (merlin-northern) [reply 02.Overview/12.Taxonomy/docs.md]: fixed

**Pull request merged**